<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mockBoard</class>
 <widget class="QWidget" name="mockBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="mock_tool_bar_container">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="mockActions">
       <property name="styleSheet">
        <string notr="true">#mockActions {padding: 0px;}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="mock_server_menu"/>
        </item>
        <item>
         <widget class="QPushButton" name="mock_server_selected_btn">
          <property name="toolTip">
           <string>Selected Mock Server</string>
          </property>
          <property name="toolTipDuration">
           <number>0</number>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">#mock_server_selected_btn{
    border: none;
    font-weight: bold;
    font-size: 14px;
}

#mock_server_selected_btn:hover{
    background: transparent;
    color: #ffffff;
}
</string>
          </property>
          <property name="text">
           <string> PushButton </string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../assets/images/icons/dropdown.ico</normaloff>../assets/images/icons/dropdown.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="mockTopActionsFrame">
          <property name="styleSheet">
           <string notr="true">QToolButton {
  border-radius: 0px;
  margin: 0px;
  padding: 10px 10px;
  font-size: 14px;
}
</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="server_status_btn">
             <property name="toolTip">
              <string>Mock Environment Setting</string>
             </property>
             <property name="text">
              <string>Play</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../assets/images/icons/play.ico</normaloff>../assets/images/icons/play.ico</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>21</width>
               <height>21</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonIconOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="mock_toggle_endpoints_btn">
             <property name="toolTip">
              <string>Toggle Endpoint List</string>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Menu</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../assets/images/icons/view.ico</normaloff>../assets/images/icons/view.ico</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>21</width>
               <height>21</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="endpointViewPanel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="endpoint_container" stretch="0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="endpoints_placeholder"/>
          </item>
          <item>
           <widget class="QFrame" name="mockactionlayout">
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="add_mock_btn">
               <property name="toolTip">
                <string>Add Mock</string>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string> Add</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../assets/images/icons/add.ico</normaloff>../assets/images/icons/add.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="import_mock_btn">
               <property name="toolTip">
                <string>Import Mock</string>
               </property>
               <property name="text">
                <string> Import</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../assets/images/icons/import.ico</normaloff>../assets/images/icons/import.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="log_view_btn">
               <property name="toolTip">
                <string>View Log</string>
               </property>
               <property name="toolTipDuration">
                <number>0</number>
               </property>
               <property name="text">
                <string>Log</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../assets/images/icons/log.ico</normaloff>../assets/images/icons/log.ico</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="mock_content_view">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="mock_right_size"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="mock_content_log">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
